### ISSUE TITLE
[Pimcore 5] Localized Fields not working anymore in Backend

### ISSUE THREAD

**Issue by dpfaffenbauer** [pimcore/pimcore#1871]
Trying to create localized fields in backend object class editor, as soon as I add the localized field, I get this error:

```
localizedfields.js?_dc=98:34 Uncaught TypeError: Cannot set property 'name' of null
    at klass.initialize (localizedfields.js?_dc=98:34)
    at new klass (prototype-light.js?_dc=98:72)
    at constructor.addDataChild (class.js?_dc=98:890)
    at constructor.handler (prototype-light.js?_dc=98:261)
    at Object.callback (ext-all-debug.js?_dc=98:8174)
    at constructor.onClick (ext-all-debug.js?_dc=98:196870)
    at constructor.onClick (ext-all-debug.js?_dc=98:197965)
    at constructor.fire (ext-all-debug.js?_dc=98:19779)
    at constructor.fire (ext-all-debug.js?_dc=98:31904)
    at constructor.publish (ext-all-debug.js?_dc=98:31880)
```

When I right click on the new localized field, I get this error:

```
class.js?_dc=98:486 Uncaught TypeError: Cannot read property 'type' of undefined
    at klass.getRestrictionsFromParent (class.js?_dc=98:486)
    at klass.getDataMenu (class.js?_dc=98:298)
    at prototype-light.js?_dc=98:261
    at klass.onTreeNodeContextmenu (class.js?_dc=98:428)
    at constructor.<anonymous> (prototype-light.js?_dc=98:261)
    at constructor.fire (ext-all-debug.js?_dc=98:19779)
    at constructor.doFireEvent (ext-all-debug.js?_dc=98:20685)
    at constructor.doFireEvent (ext-all-debug.js?_dc=98:63295)
    at constructor.prototype.doFireEvent (ext-all-debug.js?_dc=98:53639)
    at constructor.fireEventArgs (ext-all-debug.js?_dc=98:20539)
```

I am not sure why this happens though.

**Comment by dpfaffenbauer** [comment]
PS: I am on latest build and happens on Pimcore 5 demo as well (demo-basic.pimcore.org)

**Comment by weisswurstkanone** [comment]
build 99

**Comment by dpfaffenbauer** [comment]
that was quick, thank you 👍

**Comment by aarongerig** [comment]
Same goes for CRM fields (tested on https://demo-basic.pimcore.org)...

When creating the field.
```
Uncaught TypeError: Cannot read property 'name' of null
    at klass.initialize (script-proxy?scripts=minified_javascript_core_20933cd5ed0d7f00acd75a14143df722.js&_dc=99:51586)
    at new klass (prototype-light.js?_dc=99:72)
    at g.addDataChild (script-proxy?scripts=minified_javascript_core_20933cd5ed0d7f00acd75a14143df722.js&_dc=99:43984)
    at E.handler (prototype-light.js?_dc=99:261)
    at Object.callback (ext-all.js?_dc=99:22)
    at E.onClick (ext-all.js?_dc=99:22)
    at E.onClick (ext-all.js?_dc=99:22)
    at E.fire (ext-all.js?_dc=99:22)
    at E.fire (ext-all.js?_dc=99:22)
    at E.publish (ext-all.js?_dc=99:22)
```

Right click on the field.
```
Uncaught TypeError: Cannot read property 'type' of undefined
    at klass.getRestrictionsFromParent (script-proxy?scripts=minified_javascript_core_20933cd5ed0d7f00acd75a14143df722.js&_dc=99:43580)
    at klass.getDataMenu (script-proxy?scripts=minified_javascript_core_20933cd5ed0d7f00acd75a14143df722.js&_dc=99:43392)
    at prototype-light.js?_dc=99:261
    at klass.onTreeNodeContextmenu (script-proxy?scripts=minified_javascript_core_20933cd5ed0d7f00acd75a14143df722.js&_dc=99:43522)
    at E.<anonymous> (prototype-light.js?_dc=99:261)
    at E.fire (ext-all.js?_dc=99:22)
    at E.doFireEvent (ext-all.js?_dc=99:22)
    at E.doFireEvent (ext-all.js?_dc=99:22)
    at E.a.doFireEvent (ext-all.js?_dc=99:22)
    at E.fireEventArgs (ext-all.js?_dc=99:22)
```

**Comment by weisswurstkanone** [comment]
as stated above please update to build 99 :-)

**Comment by aarongerig** [comment]
Oh I did that already before posting, the error is still occurring.

**Comment by weisswurstkanone** [comment]
Good point, thx! Fixed in build #100.

**Comment by aarongerig** [comment]
Thank you 👍
